### DISCUSSION TITLE
output log json as stdout optional and not only as file

### DISCUSSION THREAD

**Discussion by arpu** [zeux/meshoptimizer#793] [Ideas]
Hello,

for automatic optimiation pieline would be nice to get the json output as stdout and not only as file

**Comment by zeux** [comment]
Are you running the pipeline on Linux or Windows? On Linux I think you can use /dev/stdout as a file name. On Windows I think I'd need to add support for `-` pseudo file name or something along these lines because `CON` doesn't play nice with piping.

**Reply by arpu** [reply]
yes nice works just fine on Linux with ```-r /dev/stdout```
